src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java

```diff
@@ -123,11 +123,16 @@ public ApexManifestFileReader(boolean includeTriggers) {
 	 */
 	private void insertIntoTestClassesArray(String strLine, ArrayList<String> testClassList){
 		String tempTestClassId = null;
-		Map<String, String> namespaceAndName = new HashMap<String, String>();
-		namespaceAndName.put("name",strLine);				
+		String[] splitByPeriod = strLine.split("\\.", 2);
+		String namespace = null;
+		String classname = strLine;
+
+		if (splitByPeriod.length == 2) {
```

Comment:
Salesforce's namespace cannot contain periods .. it's not like a Java package that can be infinitely long